cli/cmd/tapRunner.go

```diff
@@ -269,7 +270,7 @@ func portForwardApiPod(ctx context.Context, kubernetesProvider *kubernetes.Provi
 				}
 			}
 
-		case <-time.After(25 * time.Second):
+		case <- timeAfter:
```

Comment:
this is the main fix for CPU leaks, right?

Reply:
The main issue was fixed in [ikisocket itself](https://github.com/antoniodipinto/ikisocket/pull/15/files), I suspect that time.After call here also causes a leak, if it does its a much smaller one